wwwroot/data/regionMapping.json

```diff
@@ -51,14 +48,60 @@
             ],
             "nameProp": "SA2_NAME11"
         },
+        "SA1_2016": {
```

Comment:
Should this be `SA1` and the current SA1 renamed to `SA1_2011`?  I'm not sure if it matters much, but it seems like the latest data should have the best name.